benefits/enrollment/templates/enrollment/success.html

```diff
@@ -7,26 +7,16 @@
   {{ block.super |add:" enrollment-success" }}
 {% endblock classes %}
 
-{% block paragraph_content %}
-  {% if "logged-out" in page.classes %}
-    <img class="success-image" src="{% static "img/icon/happybus.svg" %}" alt="{% translate "core.icons.happybus" context "image alt text" %}">
```

Comment:
I think that, with lines 11-12 removed, the `logged-out` class is not being used anywhere.

Previously, the screen upon logout looked like
![image](https://user-images.githubusercontent.com/25497886/193727196-6c72c4f6-3a47-431d-bce4-fb43e810d2a2.png)

Now it looks like
![image](https://user-images.githubusercontent.com/25497886/193726878-d1dc2637-64eb-4ced-a0c0-fb574e312dc2.png)

Maybe in the logged out state, we can remove the media items from the page? Or something to get the same net effect.

Reply:
https://github.com/cal-itp/benefits/pull/997#discussion_r987120711

Reply:
Fixed in [6f0db24](https://github.com/cal-itp/benefits/pull/997/commits/6f0db241880135aa4b734b64776173dc483c54c0)